neuro_extras/main.py

```diff
@@ -177,6 +178,35 @@ async def launch(
         return job
 
 
+class DataCopier:
+    def __init__(self, client: neuro_api.Client):
+        self._client = client
+
+    async def launch(
+        self, storage_uri: URL, unpack: bool, src_uri: URL, dst_uri: URL,
+    ) -> neuro_api.JobDescription:
+        logger.info("Submitting a copy job")
+        copier_container = await self._create_copier_container(
+            storage_uri, unpack, src_uri, dst_uri
+        )
+        job = await self._client.jobs.run(copier_container, life_span=60 * 60)
+        logger.info(f"The copy job ID: {job.id}")
+        return job
+
+    async def _create_copier_container(
+        self, storage_uri: URL, unpack: bool, src_uri: URL, dst_uri: URL,
+    ) -> neuro_api.Container:
+        args = f"{str(src_uri)} {str(dst_uri)}"
+        if unpack:
+            args = f"-u {args}"
```

Comment:
hmm, I don't like `-u`... feels like `--upload` or `--user`. Let's follow `tar`'s terminology and rename `--unpack` to `--extract/-x` ?

Reply:
I will change in this PR so that `-u` is not released.

Reply:
I agree with `--extract/-x`

Reply:
done in 4f7df7c